prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -139,7 +145,7 @@ elements:
   - name: standard.ordering_provider_phone_number
     csvField: Ordering_provider_phone
 
-  - name: standard.ordered_test_code
+  - name: standard.test_performed
```

Comment:
PDI was using "ordered_test_code" but AZ was using "test_performed".  Turns out they are the same ValueSet.   I think we can remove ordered-test_code.